lib/ansible/playbook/task_include.py

```diff
@@ -68,7 +68,7 @@ def load(data, block=None, role=None, task_include=None, variable_manager=None,
         if bad_opts and task.action in ('include_tasks', 'import_tasks'):
             raise AnsibleParserError('Invalid options for %s: %s' % (task.action, ','.join(list(bad_opts))), obj=data)
 
-        if not task.args.get('_raw_params'):
+        if not task.args.get('_raw_params') and 'file' in task.args:
             task.args['_raw_params'] = task.args.pop('file')
```

Comment:
not a blocker, but minor optimization, why not just add None to pop? avoids extra dict access

Reply:
:+1: